social_media_pinterest/README.rst

```diff
@@ -0,0 +1,83 @@
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Pinterest Social Media Icon Extension for blog
+==============
+
+This module adds Pinterest social media icon. By default, the icon will
+be shown on the footer and about us, like the original icons.
+
+
+
+Configuration
+=============
+
+- Go in the backend under settings / website
+- You can now also add your Pinterest Account
+
+
+Usage
+=====
+
+- Normal social icons:
```

Comment:
you can use two colons here to have the following indented paragraph(s) formatted as code